BookingsApi/BookingsApi.DAL/Queries/GetHearingsForTodayQueryHandler.cs

```diff
@@ -27,6 +27,7 @@ public async Task<List<VideoHearing>> Handle(GetHearingsForTodayQuery query)
                 .Include(x => x.Participants).ThenInclude(x => x.CaseRole)
                 .Include(x => x.Participants).ThenInclude(x => x.HearingRole).ThenInclude(x => x.UserRole)
                 .Include(x => x.HearingCases).ThenInclude(x => x.Case)
+                .Include(x => x.Endpoints)
```

Comment:
Good catch, when we get a minute we should consider auto-including certain properties so we don't have to manually include it every time.